uniswap_swap_detection/src/constant.ts

```diff
@@ -0,0 +1,36 @@
+import { ethers, getEthersProvider } from "forta-agent";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const alchemyApiKey = process.env.ALCHEMY_API_KEY;
+export const UNISWAP_FACTORY_ADDRESS =
+  "0x1F98431c8aD98523631AE4a59f267346ea31F984";
+export const SWAP_EVENT =
+  "event Swap(address indexed sender,address indexed recipient,int256 amount0,int256 amount1,uint160 sqrtPriceX96,uint128 liquidity,int24 tick)";
+
+export const CREATED_POOL =
+  "event PoolCreated(address indexed token0,address indexed token1,uint24 indexed fee,int24 tickSpacing,address pool)";
+export const FUNCTION_EXACT =
+  " function exactInputSingle(address,address,uint24,address,uint256,uint256,uint256,uint160)";
```

Comment:
I don't see these being used, so should be removed. I do see they're used in the test suite, so they should be declared in the test suite file.

As a rule of thumb, only variables used in production should be in the production files. If you are declaring variables (or constants) to be used in the test suite, you should declare them there. If they're in the production files, it makes it confusing to read through because to us as reviewers, it appears like they were leftover and need to be removed.

Reply:
created pool is used in my test file

Reply:
Then it should be in the test suite file and not in the `constants.ts` file where it currently is.